src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentController.java

```diff
@@ -48,11 +48,16 @@ public AppointmentResponseModel updateAppointmentById(@RequestBody AppointmentRe
         return appointmentService.updateAppointmentDetails(appointmentRequestModel, appointmentId);
     }
 
-    @PutMapping("/{appointmentId}/status")
+    @PutMapping("/{appointmentId}/cancelled")
```

Comment:
Why did you replace status with cancelled for the endpoint?

Reply:
because I had another PUT method that did the same ting and I had to distinguish them from each other.